_integration-schemas/trello/checklists.md

```diff
@@ -0,0 +1,561 @@
+---
+tap: "trello"
+version: "1"
+key: ""
+
+name: "checklists"
+doc-link: "https://developer.atlassian.com/cloud/trello/rest/#api-checklists-id-get"
+singer-schema: "https://github.com/singer-io/tap-trello/blob/master/tap_trello/schemas/checklists.json"
+description: |
+  The {{ table.name }} table contains information about your checklists in your {{ integration.display_name }} account.
+
+replication-method: "Key-based Incremental"
```

Comment:
The code says this is full table:

```python
class Checklists(ChildStream):
    stream_id = "checklists"
    stream_name = "checklists"
    endpoint = "/boards/{}/checklists"
    key_properties = ["id"]
    replication_method = "FULL_TABLE"
    parent_class = Boards
    params = {'fields': 'all', 'checkItem_fields': 'all'}
```

https://github.com/singer-io/tap-trello/blob/master/tap_trello/streams.py#L379